package/MDAnalysis/core/universe.py

```diff
@@ -1231,6 +1233,14 @@ def add_bonds(self, values, types=None, guessed=False, order=None):
         """
         self._add_topology_objects('bonds', values, types=types,
                                  guessed=guessed, order=order)
+        self._invalidate_bond_related_caches()
+
+    def _invalidate_bond_related_caches(self):
+        """
+        Invalidate caches related to bonds and fragments.
```

Comment:
Maybe add some extra information here for the next developer?

i.e. something like:

```suggestion
        Invalidate caches related to bonds and fragments.
        
        This should be called whenever the Universe's bonds are modified.
```

Reply:
Sure, thanks!